disableKillSwitch.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+echo 'This will reset all previous rules are you sure that you want to proceed?'
+sudo ufw reset
+sudo ufw default deny incoming
+sudo ufw default allow outgoing
+sudo ufw enable
```

Comment:
One last thing, I realised that you use ufw here while you used iptables before, is it normal ?
Also maybe it would be better to use iptable, because people almost always have iptables but not always ufw ;)

Reply:
You are absolutely right! This was part of the first version of my killswitch which didn't back up old rules. Honestly I just forgot about this file. I'm going to remove it.